ibm/provider/provider.go

```diff
@@ -357,12 +357,6 @@ func Provider() *schema.Provider {
 			"ibm_iam_trusted_profiles":              iamidentity.DataSourceIBMIamTrustedProfiles(),
 			"ibm_iam_trusted_profile_policy":        iampolicy.DataSourceIBMIAMTrustedProfilePolicy(),
 
-			//backup as Service
```

Comment:
All these changes in the file is removing the support for other services which were added recently

Reply:
Take the latest code and add your changes on the top of it

Reply:
Fixed